extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1870,4 +1873,30 @@ export class AmpStoryPage extends AMP.BaseElement {
   isAutoAdvance() {
     return this.advancement_.isAutoAdvance();
   }
+
+  /**
+   * Set the data-orig-tabindex to the default tabindex of tabbable elements
+   */
+  initializeTabbableElements_() {
+    scopedQuerySelectorAll(this.element, Selectors.ALL_TABBABLE).forEach(
```

Comment:
There is a utility in modal.js that might be a helpful reference for `ALL_TABABLE`:
https://github.com/ampproject/amphtml/blob/master/src/modal.js#L106-L141

Not sure if that's overdoing it though.

Reply:
Nice find! I tried to use this method to set `tabindex="-1"` on iframes (eg: `amp-twitter`) and it wasn't reliable (when the extension would load later or the components were on a page with distance > 2), so I assume that using this list could result in other things being affected that we don't control directly. I'd say to be safe, that we should make the selector only toggle links for now and add what we want to toggle on the selector